frontend/src/App.jsx

```
@@ -1,27 +1,38 @@
     <div className="min-h-screen flex flex-col justify-between">
-      <Nav/>
-        <Routes>
-          <Route path={BASE_URL+""} element={<Home/>} />
-          <Route path={BASE_URL+"category/:category/:page"} element={<Products/>} />
-          <Route path={BASE_URL+"category/:category"} element={<Products/>} />
-          <Route path={BASE_URL+"product/:productID"} element={<ProductDisplay/>} />
-        </Routes>
-      <Footer/>
+      <Nav />
+      <Routes>
+        <Route path={BASE_URL + ""} element={<Home />} />
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)